diff -r -u gocl-0.1.4.orig/examples/Makefile.am gocl-0.1.4/examples/Makefile.am
--- gocl-0.1.4.orig/examples/Makefile.am	2013-08-03 11:08:50.000000000 +0200
+++ gocl-0.1.4/examples/Makefile.am	2013-08-19 10:27:31.455128153 +0200
@@ -39,7 +39,7 @@
 
 # gaussian-blur
 gaussian_blur_CFLAGS = $(AM_CFLAGS) $(COGL_CFLAGS)
-gaussian_blur_LDADD = $(AM_LIBS) $(COGL_LIBS) -lGL
+gaussian_blur_LDADD = $(AM_LIBS) $(COGL_LIBS) -lGL -lm
 gaussian_blur_SOURCES = gaussian-blur.c
 
 endif
Nur in gocl-0.1.4/examples: .Makefile.am.swp.
